src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -112,7 +112,7 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
-            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (self::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($file));
```

Comment:
@stof: this is to demonstrate the error handling. Please read the commit messages.